cni-plugin/main.go

```diff
@@ -134,9 +133,10 @@ func cmdAdd(args *skel.CmdArgs) error {
 	logrus.Debug("linkerd-cni: cmdAdd, parsing config")
 	conf, err := parseConfig(args.StdinData)
 	if err != nil {
+		logrus.Errorf("error parsing config: %e", err)
 		return err
 	}
-	configureLogging(conf.LogLevel)
+	configureLoggingLevel(conf.LogLevel)
```

Comment:
I know it's not part of this change, but I never understood why we logged on L133 and then configured the logging a few lines below it. Surely we should just configure it and then log? Would it make sense to just move this to the top of the function's scope now?

Reply:
There's a chicken-and-egg situation here, you can't change the logging level until you parse the config and the debug log message on L133 exists to give us a hint that we're about to parse the config. We can remove the whole line, in my opinion.

Reply:
Yeah, let's just remove it maybe.